### DISCUSSION TITLE
How can I make the editor Read Only?

### DISCUSSION THREAD

**Discussion by enesbala5** [umaranis/svelte-lexical#62] [Q&A] [answered]
I want to have the editor toolbar be hidden, and only show the content I fetch from the DB. What is the best way to do this?

**Comment by umaranis** [comment, marked as answer]
Hi @enesbala5 ,
Please look at the playground demo. It shows how you can construct the editor from individual components to your liking.

https://github.com/umaranis/svelte-lexical/blob/master/demos/playground/src/RichTextComposer.svelte
